core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -108,7 +108,8 @@ class KafkaRaftManager[T](
   time: Time,
   metrics: Metrics,
   threadNamePrefixOpt: Option[String],
-  val controllerQuorumVotersFuture: CompletableFuture[util.Map[Integer, AddressSpec]]
+  val controllerQuorumVotersFuture: CompletableFuture[util.Map[Integer, AddressSpec]],
+  apiVersions: ApiVersions
```

Comment:
We pass this same ApiVersions to ControllerServer (and QuroumController) so that we can inspect the supported features of other quorum peers.